app/cosmo/base.toml

```diff
@@ -519,7 +519,7 @@ description = "U.2 Sharkfin A hot swap controller"
 power = { rails = [ "V12_U2A_A0", "V3P3_U2A_A0" ], pmbus = false }
 sensors = { voltage = 2, current = 2 }
 name = "sharkfin_a_hsc"
-refdes = "J200"
+refdes = ["J200", "U1A"]
```

Comment:
Nit: this should just be `U1` (also true for other HSCs).  On the schematic, the physical U1 chip is split into U1A, U1B, U1C symbols, but that's just for organization.